### PR TITLE
doc/README.md: initial content

### PR DESCRIPTION
**Pull request by rs-fabrica**
Including initial content for doc/README.md

Signed-off-by: Rodrigo Severo <rodrigo@fabricadeideias.com>

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Ceph - a scalable distributed storage system
```
Comment (tchaikov) [inline doc/README.md]: the major part of this document is literally copied from https://github.com/ceph/ceph/blob/master/README.md . how are we going to structure them without replicating the contents?

**Comment by sebastian-philipp** [comment]
just wondering, shouldn't we include this documentation into the generated documentation? Like adding this to https://github.com/ceph/ceph/blob/master/doc/dev/documenting.rst ?

**Comment by zdover23** [comment]
@sebastian-philipp I'm willing to move this content to documenting.rst, if you like.

**Comment by sebastian-philipp** [comment]
@LenzGr do you remember if we had any clear preferences between readmes and the doc?

**Comment by rs-fabrica** [comment]
Please let me know if I can help.

**Comment by sebastian-philipp** [comment]
Thing is, this PR mainy duplicates https://docs.ceph.com/docs/master/start/documenting-ceph 

Maybe we should make the page easier to find and fix bugs in it, instead of adding a markdown file.

File: doc/README.md (hunk 1 above)
Comment (rs-fabrica) [reply doc/README.md:1]: You are right. But is it a problem?

The info I included here is completely relevant to the material available in this directory.

The fact that this info is also available in another place shouldn't prevent it to be available exactly where it is most relevant, should it?

Maybe, for the sake of avoiding repetition, we should keep this info here and in https://github.com/ceph/ceph/blob/master/README.md we would just point here for more info building the documentation. Does it seems reasonable?

**Comment by rs-fabrica** [comment]
jenkins retest this please

**Pull request closed without merging**